src/Libraries/PythonNodeModelsWpf/PythonStringNode.cs

```diff
@@ -72,6 +73,7 @@ private void UpdateToPython2Engine(object sender, EventArgs e)
         private void UpdateToPython3Engine(object sender, EventArgs e)
         {
             pythonStringNodeModel.Engine = PythonEngineVersion.CPython3;
+            pythonStringNodeModel.OnNodeModified();
```

Comment:
I see this is being done for `PythonStringNode` but I think `PythonNode` has similar handlers. Please check.